### PR TITLE
Added Salesforce Mass Editor

### PR DESCRIPTION
**Pull request by xgeek-net**
Could you review and marge this pull request?
Thanks!
@mailtoharshit

### CODE REVIEW

**Comment by hleb-kastseika** [comment]
@xgeek-net, I see that you forgot to change link in Google Chrome icon and now it's linked to "Salesforce Dev Tools", not to "Salesforce Mass Editor". Please, fix it

File: README.md [hunk 1]
```diff
@@ -72,3 +72,9 @@ Salesforce Dev Tools is a simple but powerful chrome extension for doing the bel
 ・Quick access to new record page, list page and object setting page.<br />
 
 [<img src="https://cloud.githubusercontent.com/assets/5466487/20777731/42582aee-b7ac-11e6-81f6-5d58afb372e9.png" align="center" width="400">](https://chrome.google.com/webstore/detail/salesforce-dev-tools/ehgmhinnhggigkogkbhnbodhbfjgncjf)
+
+## Salesforce Mass Editor&nbsp;&nbsp;[<img src="https://raw.githubusercontent.com/alrra/browser-logos/master/chrome/chrome_24x24.png"  >](https://chrome.google.com/webstore/detail/salesforce-developer-tool/fiaakhiohminpblhmlihfcdhclmphjcd?utm_source=chrome-app-launcher-info-dialog)
```
Comment (hleb-kastseika) [inline README.md]: Link in icon is linked to "Salesforce Dev Tools", not to "Salesforce Mass Editor"

**Comment by lin8898** [comment]
@last-khajiit 
Oh, exactly. I've fixed it as you pointed out, could you review this PR please? 
Thanks!

**Review by hleb-kastseika — APPROVED** [review]

**Comment by hleb-kastseika** [comment]
@lin8898 it looks fine for me now, thanks!

**Comment by xgeek-net** [comment]
Hi, @last-khajiit

Could you marge this PR please?

 @mailtoharshit

**Comment by hleb-kastseika** [comment]
Hi, @xgeek-net , unfortunately I don't have such permissions for this repo. You need to ask owner of repository for it.

**Comment by xgeek-net** [comment]
Hi, @mailtoharshit , Could you marge this?

**Comment by hleb-kastseika** [comment]
@xgeek-net seems, only owner of this repository (mailtoharshit) can do that. I asked him about this in Gitter chat

**Pull request merged**